test/unit/units/reboot_test.py

```diff
@@ -101,20 +86,9 @@ def test_main_force_reboot(
             '../data/migration-config.yml'
         main()
         assert mock_info.called
+        print(mock_Command_run.call_args_list)
```

Comment:
s/print()/

Reply:
urks, a left over, thanks for catching this

Reply:
fixed